source/support.tex

```diff
@@ -1211,31 +1211,52 @@
 
   using uintmax_t      = @\textit{unsigned integer type}@;
   using uintptr_t      = @\textit{unsigned integer type}@; // optional
-} // namespace std
-\end{codeblock}
+}
 
-\pnum
-The header also defines numerous macros of the form:
+#define INT@\placeholdernc{N}@_MIN @\seebelow@
+#define INT@\placeholdernc{N}@_MAX @\seebelow@
+#define UINT@\placeholdernc{N}@_MAX @\seebelow@
 
-\begin{codeblock}
-  INT_[FAST LEAST]{8 16 32 64}_MIN
-  [U]INT_[FAST LEAST]{8 16 32 64}_MAX
-  INT{MAX PTR}_MIN
-  [U]INT{MAX PTR}_MAX
-  {PTRDIFF SIG_ATOMIC WCHAR WINT}{_MAX _MIN}
-  SIZE_MAX
-\end{codeblock}
+#define INT_LEAST@\placeholdernc{N}@_MIN @\seebelow@
+#define INT_LEAST@\placeholdernc{N}@_MAX @\seebelow@
+#define UINT_LEAST@\placeholdernc{N}@_MAX @\seebelow@
 
-plus function macros of the form:
+#define INT_FAST@\placeholdernc{N}@_MIN @\seebelow@
+#define INT_FAST@\placeholdernc{N}@_MAX @\seebelow@
+#define UINT_FAST@\placeholdernc{N}@_MAX @\seebelow@
 
-\begin{codeblock}
-  [U]INT{8 16 32 64 MAX}_C
+#define INTPTR_MIN @\seebelow@
+#define INTPTR_MAX @\seebelow@
+#define UINTPTR_MAX @\seebelow@
+
+#define INTMAX_MIN @\seebelow@
+#define INTMAX_MAX @\seebelow@
+#define UINTMAX_MAX @\seebelow@
+
+#define PTRDIFF_MIN @\seebelow@
+#define PTRDIFF_MAX @\seebelow@
+#define SIG_ATOMIC_MIN @\seebelow@
+#define SIG_ATOMIC_MAX @\seebelow@
+#define SIZE_MAX @\seebelow@
+
+#define WCHAR_MIN @\seebelow@
+#define WCHAR_MAX @\seebelow@
+#define WINT_MIN @\seebelow@
+#define WINT_MAX @\seebelow@
+
+#define INT@\placeholdernc{N}@_C(value) @\seebelow@
+#define UINT@\placeholdernc{N}@_C(value) @\seebelow@
+
+#define INTMAX_C(value) @\seebelow@
+#define UINTMAX_C(value) @\seebelow@
 \end{codeblock}
 
 \pnum
 The header defines all types and macros the same as
```

Comment:
@zygoloid: updated version pushed that lists the four sizes that we support explicitly.